### PR TITLE
[Docs] E2E Preparation for Beta TestNet release

### PR DESCRIPTION
**Pull request by Olshansk**
Miscleanious improvements of the docs for the beta TestNet release.

### CODE REVIEW

File: docusaurus/docs/explore/rpc.md [hunk 1]
```diff
@@ -3,17 +3,31 @@ title: RPC Endpoints
...
+  - [Alpha RPC Endpoints](#alpha-rpc-endpoints)
+  - [Alpha JSON-RPC Example](#alpha-json-rpc-example)
 
-We have provided `gRPC`, `JSON-RPC` and `REST` endpoints, which are available here:
+## Types of RPC Endpoints
 
-- `https://shannon-testnet-grove-seed-rpc.poktroll.com`
-- `https://shannon-testnet-grove-seed-grpc.poktroll.com`
-- `https://shannon-testnet-grove-seed-api.poktroll.com`
+We have provided `gRPC`, `JSON-RPC` and `REST` endpoints, which are available here:
```
Comment (red-0ne) [inline docusaurus/docs/explore/rpc.md:16]:
```suggestion
```

File: docusaurus/docs/explore/rpc.md [hunk 2]
```diff
@@ -3,17 +3,31 @@ title: RPC Endpoints
...
 
 You can review the difference between them in the [Cosmos SDK docs](https://docs.cosmos.network/main/learn/advanced/grpc_rest#comparison-table).
 
-### JSON-RPC
+## Beta TestNet
+
+### Beta RPC Endpoints
+
+We have provided `gRPC`, `JSON-RPC` and `REST` endpoints, which are available here:
```
Comment (red-0ne) [inline docusaurus/docs/explore/rpc.md:24]:
```suggestion
We provide `gRPC`, `JSON-RPC` and `REST` endpoints, which are available here:
```

File: docusaurus/docs/explore/rpc.md [hunk 3]
```diff
@@ -27,10 +41,26 @@ Using the `poktrolld` binary:
 poktrolld query block --type=height 0 --node https://shannon-testnet-grove-seed-rpc.poktroll.com
 ```
 
-### gRPC
+## Alpha TestNet
+
+### Alpha RPC Endpoints
 
-_TODO_TECHDEBT: Add a gRPC example_
+We have provided `gRPC`, `JSON-RPC` and `REST` endpoints, which are available here:
```
Comment (red-0ne) [inline docusaurus/docs/explore/rpc.md:48]:
```suggestion
We provide `gRPC`, `JSON-RPC` and `REST` endpoints, which are available here:
```

File: docusaurus/docs/explore/roadmap.md [hunk 4]
```diff
@@ -3,42 +3,62 @@ title: Roadmap
...
-          Beta TestNet 1 <br> ------- <br> Kickoff Incentivized TestNet:
-          Beta TestNet 2 <br> ------- <br> Shannon Migration:
-                Start wPOKT migration:
-                Kickoff audits
+      Alpha TestNet(s) <br> (Q3/Q4):
+	      ✅ Shannon Foundation:
+	      ✅ Shannon SDK:
+	      ✅ Relay Mining:
+	      ✅ Morse Utility Parity:
+	      ✅ Token Logic Modules:
```
Comment (okdas) [inline docusaurus/docs/explore/roadmap.md:24]: Love the green!

File: docusaurus/docs/explore/rpc.md [hunk 5]
```diff
@@ -27,10 +41,26 @@ Using the `poktrolld` binary:
...
 
-### REST
+- **RPC**: `https://shannon-testnet-grove-seed-rpc.alpha.poktroll.com`
```
Comment (okdas) [inline docusaurus/docs/explore/rpc.md:50]: These are not active currently, but soon will be - I've already added the changes. Need to apply/test.

File: docusaurus/docs/explore/rpc.md [hunk 6]
```diff
@@ -27,10 +41,26 @@ Using the `poktrolld` binary:
...
+Using `curl`:
+
+```bash
+curl -X POST https://shannon-testnet-grove-seed-rpc.poktroll.com/block
+```
+
+Using the `poktrolld` binary:
+
+```bash
+poktrolld query block --type=height 0 --node https://shannon-testnet-grove-seed-rpc.poktroll.com
```
Comment (red-0ne) [inline docusaurus/docs/explore/rpc.md]:
```suggestion
poktrolld query block --type=height 1 --node https://shannon-testnet-grove-seed-rpc.poktroll.com
```

Height `0` returns an error

File: docusaurus/docs/explore/rpc.md [hunk 7]
```diff
@@ -27,10 +41,26 @@ Using the `poktrolld` binary:
...
+- **gRPC**: `https://shannon-testnet-grove-seed-grpc.alpha.poktroll.com`
+- **REST**: `https://shannon-testnet-grove-seed-api.alpha.poktroll.com`
 
-_TODO_TECHDEBT: Add a REST example_
+### Alpha JSON-RPC Example
+
+Using `curl`:
+
+```bash
+curl -X POST https://shannon-testnet-grove-seed-rpc.poktroll.com/block
```
Comment (okdas) [inline docusaurus/docs/explore/rpc.md:59]:
```suggestion
curl -X POST https://shannon-testnet-grove-seed-rpc.alpha.poktroll.com/block
```

File: docusaurus/docs/explore/rpc.md (hunk 6 above)
Comment (okdas) [inline docusaurus/docs/explore/rpc.md:65]:
```suggestion
poktrolld query block --type=height 0 --node https://shannon-testnet-grove-seed-rpc.alpha.poktroll.com
```

**Review by okdas — APPROVED** [review]
LGTM with a couple of quick notes

File: docusaurus/docs/explore/tools.md [hunk 8]
```diff
@@ -3,10 +3,24 @@ title: Tools & References
...
+
+- 🪙 [Shannon Beta Token Faucet](https://faucet.beta.testnet.pokt.network/)
+- 🗺️ [Shannon Beta Explorer](https://shannon.beta.testnet.pokt.network)
+- 🗺️ [PoktScan's Shannon Beta Explorer](https://shannon.beta.poktscan.com/)
+
+## Alpha TestNet
+
+- 🪙 [Shannon Alpha Token Faucet](https://faucet.alpha.testnet.pokt.network/)
+- 🗺️ [Shannon Alpha Explorer](https://shannon.alpha.testnet.pokt.network)
+- 🗺️ [PoktScan's Shannon Alpha Explorer](https://shannon.alpha.poktscan.com/)
```
Comment (red-0ne) [inline docusaurus/docs/explore/tools.md:20]: Those links are down.

Ignore if this is expected.

File: docusaurus/docs/explore/rpc.md [hunk 9]
```diff
@@ -27,10 +41,26 @@ Using the `poktrolld` binary:
...
+Using `curl`:
+
+```bash
+curl -X POST https://shannon-testnet-grove-seed-rpc.alpha.poktroll.com/block
+```
+
+Using the `poktrolld` binary:
+
+```bash
+poktrolld query block --type=height 0 --node https://shannon-testnet-grove-seed-rpc.alpha.poktroll.com
```
Comment (red-0ne) [inline docusaurus/docs/explore/rpc.md:65]:
```suggestion
poktrolld query block --type=height 1 --node https://shannon-testnet-grove-seed-rpc.alpha.poktroll.com
```
Using height `0` returns an error. Could you please update it for all occurrences in this doc?

**Review by red-0ne — COMMENTED** [review]
Left a few comments but I like the shape it's taking :seedling:

**Pull request merged**